virtual_rainforest/models/soil/constants.py

```diff
@@ -4,117 +4,141 @@
 """  # noqa: D205, D415
```

Comment:
Why is there a " after constants?

Reply:
I think I had it as "constants", then decided against but forgot to delete the second "